WordPress/src/main/java/org/wordpress/android/ui/prefs/notifications/NotificationsSettingsFragment.java

```diff
@@ -517,6 +518,15 @@ public boolean onPreferenceTreeClick(PreferenceScreen preferenceScreen, @NonNull
             AnalyticsTracker.track(AnalyticsTracker.Stat.NOTIFICATION_SETTINGS_DETAILS_OPENED);
         }
 
+        // Track when users enable and disable global notification setting (i.e. "Allow app notifications").
+        if (preference instanceof SwitchPreference&& preference.getKey().equalsIgnoreCase(getString(R.string.wp_pref_notification_receive))) {
```

Comment:
Minor nit: we need a space before `&&` here.

Reply:
Good eye!  That would have bothered me so much that I would have created another pull request just to fix it.  I added the space in .Good eye!  That would have bothered me so much that I would have created another pull request just to fix it.  I added the space in 85b326cbd7ec9af2d1deed0687ad7267600e4baa.